events/pull-meetup.js

```diff
@@ -47,9 +36,12 @@ function pull (opts) {
     body.results.forEach((result) => {
       const title = result.name.toLowerCase()
 
-      if (title.includes('nodeschool')) return
-      if (title.includes('mongodb') && title.includes('node')) return
-      if (title.includes('find a tech job') && title.includes('node')) return
+      if (
+        title.includes('find a tech job') ||
+        title.includes('nodeschool') ||
+        / mongodb (?:user group|meet ?up)$/.test(title) ||
```

Comment:
There are also a lot of irrelevant events like Angular Ember etc but it's kinda hard to have ONLY relevant Node.js events. This a flaw in Meetup categories and topic.